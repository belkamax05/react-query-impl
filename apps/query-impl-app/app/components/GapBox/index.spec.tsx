import { render } from '@testing-library/react';
import { GapBox } from './index';

describe('GapBox', () => {
  it('should render with default props', () => {
    const { container } = render(<GapBox />);
    expect(container.firstChild).toHaveStyle('display: flex');
    expect(container.firstChild).toHaveStyle('gap: 8px');

    expect(container).toMatchSnapshot();
  });

  it('should render with custom props', () => {
    const { container } = render(<GapBox display="grid" gap={2} />);
    expect(container.firstChild).toHaveStyle('display: grid');
    expect(container.firstChild).toHaveStyle('gap: 16px');

    expect(container).toMatchSnapshot();
  });
});
