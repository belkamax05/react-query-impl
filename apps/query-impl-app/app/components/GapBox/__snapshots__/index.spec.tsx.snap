// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`GapBox should render with custom props 1`] = `
<div>
  <div
    class="MuiBox-root css-1p44k52"
  />
</div>
`;

exports[`GapBox should render with default props 1`] = `
<div>
  <div
    class="MuiBox-root css-1i27l4i"
  />
</div>
`;
